sql/moz-fx-data-shared-prod/telemetry_derived/newtab_merino_extract_to_gcs_v2/metadata.yaml

```diff
@@ -1,6 +1,6 @@
 friendly_name: Newtab Merino BigQuery Extract to Google Cloud Storage (GCS)
 description: |-
-  Newtab engagements aggregated on scheduled_corpus_item_id are exported to a GCS bucket for Merino to consume.
+  Newtab engagements aggregated on corpus_item_id are exported to a GCS bucket for Merino to consume.
   The table rebuilds every 20 minutes and aggregate one day of data.
 owners:
```

Comment:
Perhaps you could add BE team members as owners. That way you get the notifications